jenkins/stage-maven-release.jenkinsfile

```diff
@@ -12,38 +12,31 @@ pipeline {
             alwaysPull true
         }
     }
+    triggers {
+        GenericTrigger(
+            genericVariables: [
+                [key: 'ref', value: '$.ref'],
+                [key: 'version', value: '$.ref', regexpFilter: 'refs/tags/v' ],
+            ],
+            tokenCredentialId: 'jenkins-opensearch-java-generic-webhook-token',
+            causeString: 'A tag was cut on opensearch-project/opensearch-java repository causing this workflow to run',
+            printContributedVariables: false,
+            printPostContent: false,
+            regexpFilterText: '$ref',
+            regexpFilterExpression: '^refs/tags/.*'
+        )
+    }
     environment {
-        VERSION = "${params.VERSION}"
```

Comment:
Set the version to the tag, and it will try to release that version. Otherwise by default the version has `-SNAPSHOT` in it and it doesn't work.

Reply:
Added!